shared/profile/search/bar.js

```diff
@@ -78,6 +79,7 @@ const styles = Styles.styleSheetCreate({
     },
     isElectron: {
       ...Styles.desktopStyles.clickable,
+      ...Styles.desktopStyles.windowDraggingClickable,
```

Comment:
Should we consider having `desktopStyles.clickable` just imply `desktopStyles.windowDraggingClickable` directly?

Reply:
hmmmm, i think there's side effects to doing it, so lets keep the changes minimal